extensions/amp-story-education/0.1/amp-story-education.css

```diff
@@ -20,7 +27,8 @@ amp-story-education {
   width: 100% !important;
   background: rgba(0, 0, 0, 0.8) !important;
   color: #FFF !important;
-  font-family: 'Open Sans', sans-serif !important;
+  font-family: 'Open Sans', 'Open Sans-fallback', sans-serif !important;
+  line-height: 1em !important;
```

Comment:
This will look horrible if the text adds up to two lines, do you need it? If so use something like 1.3-1.5em

Reply:
Sure `1.5em` works as well. We need any value here so that the size is maintained for the line-height (it's part of the styles required).